src/service/blockaid/index.ts

```diff
@@ -1,8 +1,15 @@
 import Blockaid from "@blockaid/client";
 import Prometheus from "prom-client";
 import { Logger } from "pino";
+import { Networks, TransactionBuilder } from "stellar-sdk";
 
 import { ERROR } from "../../helper/error";
+import { NetworkNames } from "../../helper/validate";
+
+const NetworkNameBlockaid: { [index: string]: "pubnet" | "futurenet" } = {
+  PUBLIC: "pubnet",
+  FUTURENET: "futurenet",
+};
```

Comment:
Should this include testnet? From their docs, it looked like they support testnet now

Reply:
from their docs it does look that way but their type here is `"pubnet" | "futurenet"` so I think maybe that's an open question to bring up with them.